wordpress-mobile/WordPress-iOS
# Aztec editor: Import media from other applications

This partially addresses [7081](https://github.com/wordpress-mobile/WordPress-iOS/issues/7081).

For iOS11+, this feature adds a new ellipsis ('...') option to the Visual 2.0 editor > add media toolbar by utilizing `UIDocumentPickerViewController` to enable users to add photos and video from other applications.

![v2editor_simulator](https://user-images.githubusercontent.com/1816888/31347187-bb3f9de4-acd8-11e7-8ff4-b333c93a9287.gif)

**To test:**
- Prerequisites: 
    - Any device must have external media to select from. Ex: iCloud enabled, Google Drive installed.
    - The Visual 2.0 editor must be enabled.
- Run the app on iOS11+ devices. Edit a post. Tap the add media button and verify '...' appears on the toolbar.
- Verify you can add photos and videos from a selected location.
- Run the app on <iOS11 devices. Verify '...' does not appear in the menu.

Needs review: @frosty
